$NetBSD: patch-as,v 1.2 1999/11/24 17:34:10 hubertf Exp $

Place(s) where this was (most probably...) in before:
     NetBSD: patch-as,v 1.1 1999/11/03 04:18:48 jlam Exp 

diff -x *.orig -urN ./lib/config/mxmkmf.in /usr/pkgsrc/x11/lesstif/work.i386.unpatched/lesstif-0.89.4/lib/config/mxmkmf.in
--- ./lib/config/mxmkmf.in	Fri Sep  3 01:51:36 1999
+++ /usr/pkgsrc/x11/lesstif/work.i386.unpatched/lesstif-0.89.4/lib/config/mxmkmf.in	Wed Nov 24 17:30:54 1999
@@ -7,4 +7,4 @@
 #
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-imake -DUseInstalled -I@libdir@/X11/config -I@x_libraries@/X11/config
+imake -DUseInstalled -I@libdir@/X11/lesstif -I@x_libraries@/X11/config
